jetty-server/src/main/java/org/eclipse/jetty/server/ConnectionFactory.java

```diff
@@ -125,4 +125,16 @@
          */
         Detection detect(ByteBuffer buffer);
     }
+
+    /**
+     * A ConnectionFactory that can configure the connector.
```

Comment:
Please fix the javadoc, as `Configuring` is not a `ConnectionFactory`. 
Unless, only a `ConnectionFactory` can implement `Configuring`.
If something else can, then the javadoc needs to be fixed, something like:  "Implementing this class allows to configure the connector (or its component tree) at connector startup".

Reply:
I'll make it extend ConnectionFactory like the other 'ing' inner interfaces